Plugin.pm

```diff
@@ -1365,6 +1368,11 @@ sub _albumItem {
 		$item->{line2} = $artist . ($albumYear ? ' (' . $albumYear . ')' : '');
 		$item->{name} .= $albumYear ? "\n(" . $albumYear . ')' : '';
 	}
+	
+	if ( $prefs->get('parentalWarning') && $album->{parental_warning} ) {
+		$item->{name} .= ' [E]';
+		$item->{line1} .= ' [E]';
+	}
```

Comment:
This is for display purposes only, isn't it? Eg. the suffix wouldn't be part of an album name when trying to look up MAI information?

Reply:
I'll remove that in MAI.